### DISCUSSION TITLE
Welcome to AC695N-demo-xbox360-WindowsPC_only Discussions!

### DISCUSSION THREAD

**Discussion by PING020903** [PING020903/AC695N-demo-xbox360-WindowsPC_only#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 欢迎光临该手柄项目！！
我们将讨论区作为与社区其他成员联系的场所。我们希望您：
 * 提出你想知道的问题
 * 分享想法
 * 与其他社区成员互动
 * 欢迎他人，思想开放。请记住，这是一个我们共同建设的社区
 #### 要开始学习，请在下方评论中介绍您自己，并告诉我们您在这个社区做什么。

**Comment by yue184** [comment]
我目前在基于AC695N 开发蓝牙协议栈：https://github.com/bluekitchen/btstack 这个是开源的协议栈

**Reply by PING020903** [reply]
还是请您自己看看这个 https://github.com/PING020903/AC695N-jieli--demo-GamePad/blob/main/sdk/apps/dongle/btstack_demo.c
我自己暂时还没有用过杰里蓝牙部分，帮不上忙
